### ISSUE TITLE
Please bump phpunit/php-token-stream

### ISSUE THREAD

**Issue by richardhj** [contao/core-bundle#1350]
The dependency `phpunit/php-token-stream` was added in b2b1ef91db8e076e2019cf3885e1a3a4ed94149e.

It is not possible to install `phpunit/phpunit` in version 7.
Maybe you can safely unlock `phpunit/php-token-stream` for v2.0 and v3.0.

```
  Problem 1
    - Conclusion: don't install phpunit/phpunit 7.0.0
    - Conclusion: don't install phpunit/phpunit 7.1.x-dev
    - Conclusion: remove phpunit/php-token-stream 1.4.12
    - Installation request for phpunit/phpunit ^7.0 -> satisfiable by phpunit/phpunit[7.0.0, 7.0.x-dev, 7.1.x-dev].
    - Conclusion: don't install phpunit/php-token-stream 1.4.12
    - phpunit/phpunit 7.0.x-dev requires phpunit/php-code-coverage ^6.0 -> satisfiable by phpunit/php-code-coverage[6.0.0, 6.0.1, 6.0.x-dev].
    - phpunit/php-code-coverage 6.0.0 requires phpunit/php-token-stream ^3.0 -> satisfiable by phpunit/php-token-stream[3.0.0, 3.0.x-dev].
    - phpunit/php-code-coverage 6.0.1 requires phpunit/php-token-stream ^3.0 -> satisfiable by phpunit/php-token-stream[3.0.0, 3.0.x-dev].
    - phpunit/php-code-coverage 6.0.x-dev requires phpunit/php-token-stream ^3.0 -> satisfiable by phpunit/php-token-stream[3.0.0, 3.0.x-dev].
    - Can only install one of: phpunit/php-token-stream[3.0.0, 1.4.12].
    - Can only install one of: phpunit/php-token-stream[3.0.x-dev, 1.4.12].
    - Installation request for phpunit/php-token-stream (locked at 1.4.12) -> satisfiable by phpunit/php-token-stream[1.4.12].
```

**Comment by leofeyer** [comment]
I assume this needs to be changed in Contao 4.5 only, as PHPUnit 7 requires PHP 7.1+, right?

@contao/developers Can we set the version constraint to `^1.4|^2.0|^3.0` or is the [PhpFileLoader](https://github.com/contao/core-bundle/blob/4.4/src/Config/Loader/PhpFileLoader.php#L65) class not compatible with one of the versions?

**Comment by Toflar** [comment]
No bloody idea but that's why we have unit tests :)

**Comment by leofeyer** [comment]
I don't think you can unit test different versions of the same package. At least not automated, because you would have to provide different `composer.json` files, wouldn't you?

**Comment by Toflar** [comment]
We test for lowest and highest combinations. Imho that's enough.

**Comment by leofeyer** [comment]
No, we test the **same** packages with different PHP versions. However, we would have to test **different** package versions here.

**Comment by Toflar** [comment]
We have a testing matrix? In master we currently test both, PHP 7.1 and 7.2 with both `composer update` and `composer update --prefer-lowest`. So we do test different package versions! We wouls test `1.4` and `3.0`, just not `2.0` possibly.

**Comment by leofeyer** [comment]
Fixed in ccecddf9b00acf36b7e852a9f4e17ad99a68c698.
